src/RunTests/build/Microsoft.Build.RunTests.targets

```diff
@@ -0,0 +1,12 @@
+<Project>
+  <UsingTask TaskName="Microsoft.Build.RunTestsTask" AssemblyFile="$(MSBuildThisFileDirectory)netstandard2.0\Microsoft.Build.RunTests.dll" />
+  <Target Name="RunTests" AfterTargets="MSBuildRunTests" Condition="$(IsTestProject)==true">
```

Comment:
is we want to plug a different test runner we can do it using a different `Condition` here right?

Reply:
Yes, that is correct!